released/packages/coq-smtcoq/coq-smtcoq.2.0+8.13/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+maintainer: "Chantal.Keller@lri.fr"
+homepage: "https://smtcoq.github.io/"
+dev-repo: "git+https://github.com/smtcoq/smtcoq.git"
+bug-reports: "https://github.com/smtcoq/smtcoq/issues"
+license: "CECILL-C"
+build: [
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "ocaml" {>= "4.07.1" }
+  "coq" {>= "8.13~" & < "8.14~"}
+]
+tags: [ 
+  "category:Computer Science/Decision Procedures and Certified Algorithms/Decision procedures"
+  "category:Miscellaneous/Coq Extensions"
+  "keyword: SMT"
+  "keyword: SAT"
+  "keyword: automation"
```

Comment:
```suggestion
  "keyword: automation"
  "logpath:SMTCoq"
```
to add the path to `Require` from, as suggested in https://github.com/coq/opam-coq-archive#website-and-opam-metadata